upup/pkg/fi/dryrun_target.go

```diff
@@ -49,6 +50,24 @@ type render struct {
 	changes Task
 }
 
+// ByTaskKey sorts []*render by TaskKey (type/name)
+type ByTaskKey []*render
+
+func (a ByTaskKey) Len() int      { return len(a) }
```

Comment:
A `Len()` wrapper for `len()`?

The 
> Kris wants to jump on her soap box and preach about over abstraction
engineer inside of me wants to wave some hands in the air here..

Not a deal breaker because this *could* be meaningful if we need to grow this in other cloud providers later..